erpnext/healthcare/doctype/patient/patient.py

```diff
@@ -98,7 +99,10 @@ def create_customer(doc):
 		'customer_name': doc.patient_name,
 		'customer_group': customer_group,
 		'territory' : territory,
-		'customer_type': 'Individual'
+		'customer_type': 'Individual',
+		'default_currency': doc.default_currency,
+		'default_price_ist': doc.default_price_list,
```

Comment:
We should fetch the currency and price list from the Company document if it's not set in the Patient document.